api/src/dev/server/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apk --no-cache add openjdk8
 # https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/master/alpine/Dockerfile
 #
 
-ENV CLOUD_SDK_VERSION 165.0.0
+ENV CLOUD_SDK_VERSION 216.0.0
```

Comment:
Note this sdk update ... in case you see something that is deprecated and breaks. I tried to find what version gcloud sql import csv was released out of beta into the official sdk but i could not find it.

Reply:
OK, please notify #workbench-eng.